R/RandomExtensions/Deps.toml

```diff
@@ -4,3 +4,7 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 ["0-0.1"]
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+
+["0.2.1-0"]
+SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
```

Comment:
You can have `Test` as a test-only dependency.

Reply:
Yes, I thought it was simpler like that when `Test` is not an expensive dependency to have. It's in reaction to my mistake to have put `SparseArrays` as a test-only dep previously, which was not catched by running the tests! I can change back the `Test` dep if you like.